_observing-your-data/trace/distributed-tracing.md

```diff
@@ -0,0 +1,162 @@
+
+Use distributed tracing for the following purposes:
+
+- **Optimize performance:** Identify and resolve bottlenecks, reducing latency in your applications.
+- **Troubleshoot errors:** Quickly pinpoint the source of errors or unexpected behavior in your distributed system.
+- **Allocate resources:** Optimize resource allocation by understanding usage patterns of different services.
+- **Visualize service dependencies:** Visualize dependencies between services, helping you to manage architectures. 
+
+## Distributed tracing pipeline
+
+OpenSearch provides a distributed tracing pipeline that can be used to ingest, process, and visualize tracing data with query and alerting functionality. [OpenTelemetry](https://opentelemetry.io/) is an open-source observability framework that provides a set of APIs, libraries, agents, and collectors for generating, capturing, and exporting telemetry data. The distributed tracing pipeline consists of the following components: 
+
+- **Instrumentation:** Instrumenting your application code with OpenTelemetry SDKs.
+- **Propagation:** Injecting trace context into requests as they propagate through your system.
+- **Collection:** Collecting trace data from your application.
+- **Processing:** Aggregating trace data from multiple sources and enriching it with additional metadata.
+- **Exporting:** Sending trace data to a backend for storage and analysis. 
+
+OpenSearch is often chosen as the sink for storing trace data.
+
+## Trace analytics
+
+OpenSearch provides a `trace-analytics` plugin for visualizing trace data in real time. The plugin includes pre-built dashboards for analyzing trace data, such as service maps, latency histograms, and error rates. With OpenSearch's distributed tracing pipeline, you can quickly identify bottlenecks and errors in your applications. See the [Trace analytics]({{site.url}}{{site.baseurl}}/observing-your-data/trace/index/) documentation for more information. 
+
+## Get started
+
+The distributed tracing feature is experimental as of OpenSearch 2.10. To begin using the distributed tracing feature, you need to first enable it using `opensearch.experimental.feature.telemetry.enabled` feature flag, and subsequently activate the tracer, using the dynamic setting `telemetry.tracer.enabled`. It's important to exercise caution when enabling this feature, as it can consume system resources. Detailed information on enabling and configuring distributed tracing, including on-demand troubleshooting and request sampling, is described in the following sections.
```

Comment:
```suggestion
The distributed tracing feature is experimental as of OpenSearch 2.10. To begin using the distributed tracing feature, you need to first enable it using the `opensearch.experimental.feature.telemetry.enabled` feature flag and subsequently activate the tracer, using the dynamic setting `telemetry.tracer.enabled`. It's important to exercise caution when enabling this feature because it can consume system resources. Detailed information on enabling and configuring distributed tracing, including on-demand troubleshooting and request sampling, is described in the following sections.
```